### PR TITLE
Stylesheets - Checkout new style files from PR

### PR DESCRIPTION
**Pull request by nlaz**
Pulls out the style changes from https://github.com/nlaz/pensieve/pull/6. Adds a `vars.scss` file to contain sass variables then cleans up and updates some styles.

### CODE REVIEW

File: client/assets/_items.scss [hunk 1]
```diff
@@ -1,149 +1,146 @@
...
  * Items styles
  * http://bit.ly/1ywfhq7
  */
+@import './vars';
 
-$C_borderColor: #dddddd;
 $C_borderRadius: 4px;
-$C_buttonGreen: #41d0a5;
-$C_buttonDarkGreen: #339777;
+$C_darkBorder: #d5d5d5;
```
Comment (DrewRomanyk) [inline client/assets/_items.scss]: Any reason why this isn't included?

File: client/assets/_items.scss [hunk 2]
```diff
@@ -1,149 +1,146 @@
...
-	}
+  background: none;
+  border: none;
+  border-radius: 0;
+  height: auto;
+  margin: 0;
+  padding: 0;
+
+  .button-reviewNow {
+    background: #f3f3f3;
```
Comment (DrewRomanyk) [inline client/assets/_items.scss]: Same

**Comment by DrewRomanyk** [comment]
Me gusta the tabs to spaces :P Also is there any reason to not standardize the rest of the colors?

File: client/assets/_items.scss (hunk 1 above)
Comment (nlaz) [reply client/assets/_items.scss]: Oops! Missed these in the merge 😲

**Pull request merged**